docs/use-cases/other.md

```diff
@@ -0,0 +1,7 @@
+# Other
+
+## Docs
+
+We don't have any special docs for "other". Try these:
```

Comment:
Perhaps also link to https://github.com/terrytangyuan/awesome-argo?

Reply:
Maybe in a subsequent PR? Would you like to add? This is all open for contribution.

Reply:
Sure